src/NServiceBus.AzureStoragePersistence/SagaPersisters/Azure/AzureSagaPersister.cs

```diff
@@ -111,13 +111,10 @@ public AzureSagaPersister(string connectionString, bool autoUpdateSchema)
             }
             else
             {
-                throw new NotSupportedException(
-                    string.Format("The property type '{0}' is not supported in windows azure table storage",
-                        propertyInfo.PropertyType.Name));
+                throw new NotSupportedException($"The property type '{propertyInfo.PropertyType.Name}' is not supported in windows azure table storage");
             }
 
-            var tableEntity = (await table.ExecuteQueryAsync(query).ConfigureAwait(false)).SafeFirstOrDefault();
-            return tableEntity;
+            return (await table.ExecuteQueryAsync(query).ConfigureAwait(false)).SafeFirstOrDefault();
```

Comment:
While I see the intent, I disagree with this kind of optimization. It's not something that improves readability or saves anything. Either way, what I wanted truly to say is that this is a personal preference.